md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ProcessAPI.java

```diff
@@ -784,41 +854,41 @@ RestWrapper importData(@ModelAttribute("fileString")
                 propertiesDAO.insert(insertProperties);
 
             }
-            com.wipro.ats.bdre.md.dao.jpa.Process parentProcessInserted=processDAO.returnProcess(parentProcess.getProcessCode());
-            File oldDir = new File(homeDir+"/bdre-wfd/intermediateDir");
-            File newDir=new File(homeDir+"/bdre-wfd/"+parentProcessInserted.getProcessId());
-            if (newDir.exists())
-            {
+            com.wipro.ats.bdre.md.dao.jpa.Process parentProcessInserted = processDAO.returnProcess(parentProcess.getProcessCode(),principal.getName());
+            File oldDir = new File(homeDir + "/bdre-wfd/intermediateDir");
+            File newDir = new File(homeDir + "/bdre-wfd/" + parentProcessInserted.getProcessId());
+            if (newDir.exists()) {
                 newDir.delete();
             }
-            if ( oldDir.isDirectory() ) {
+            if (oldDir.isDirectory()) {
                 oldDir.renameTo(newDir);
             } else {
                 oldDir.mkdir();
                 oldDir.renameTo(newDir);
             }
             restWrapper = new RestWrapper(processExport, RestWrapper.OK);
-        }  catch (MetadataException e) {
+        } catch (MetadataException e) {
             LOGGER.error(e);
             restWrapper = new RestWrapper(e.getMessage(), RestWrapper.ERROR);
-        }catch (IOException io){
+        } catch (IOException io) {
             LOGGER.error(io);
             restWrapper = new RestWrapper(io.getMessage(), RestWrapper.ERROR);
         }
         return restWrapper;
     }
 
     @RequestMapping(value = {"/execute", "/execute/"}, method = RequestMethod.POST)
-    @ResponseBody public
-    RestWrapper executeProcess(@ModelAttribute("process")
-                               @Valid Process process, BindingResult bindingResult, Principal principal) {
+    @ResponseBody
+    public RestWrapper executeProcess(@ModelAttribute("process")
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused method parameter(s) "bindingResult". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1172)